### PR TITLE
Remove obsolete v1 tests (part 1)

### PR DESCRIPTION
**Pull request by jason-fox**
In order to merge telefonicaid/iotagent-node-lib#995 I want to get the unit-tests running without relying on NGSI-v1. This PR just deletes tests which are already duplicated in the NGSI-v2 test suite. Any drop in coverage should be within acceptable limits

This is the ultralight equivalent of [iotagent-json/553](https://github.com/telefonicaid/iotagent-json/pull/553). Supersedes #478

### CODE REVIEW

**Comment by jason-fox** [comment]
I'm not sure what coveralls means here - I can't see any significant coverage difference:

Screenshot 2021-05-17 at 16 21 51

Running locally is also different:

Screenshot 2021-05-17 at 16 24 00

**Comment by jason-fox** [comment]
The miss on [commonBindings.js line 62](https://github.com/telefonicaid/iotagent-ul/blob/master/lib/commonBindings.js#L62) is totally expected - it is NGSI-v1 specific code.

File: test/unit/httpBinding-test.js [hunk 1]
```diff
@@ -1,694 +0,0 @@
...
-
-                    const parsedBody = JSON.parse(body);
-
-                    response.statusCode.should.equal(200);
-                    should.exist(parsedBody.transport);
-                    parsedBody.transport.should.equal('HTTP');
-                    done();
-                });
-            });
-        });
```
Comment (mapedraza) [inline test/unit/httpBinding-test.js:214]: This it section is not present on NGSI-v2 test [here](https://github.com/telefonicaid/iotagent-ul/blob/c71f1d3cb4cd4c81c51446d3f0e16c020ae1dae0/test/unit/ngsiv2/httpBindings-test.js#L169-L251)

**Comment by mapedraza** [comment]
Im going to summarize all changes needed to have the same coverage (or, at least, the same "scenarios") on V1 and V2 for an effective v1 removal from IoTA Node Lib codebase. Not all of them apply to this specific PR (since still remains some tests on V1 folder)

1. Fix tests that are not matching on V2 
    - commandsHttp-test.js ([1](https://github.com/telefonicaid/iotagent-ul/blob/master/test/unit/commandsHttp-test.js#L128) and [2](https://github.com/telefonicaid/iotagent-ul/blob/master/test/unit/commandsHttp-test.js#L169)
    - [commandsMqtt-test.js](https://github.com/telefonicaid/iotagent-ul/blob/master/test/unit/commandsMqtt-test.js#L166)
    - [httpBinding-test.js](https://github.com/telefonicaid/iotagent-ul/blob/master/test/unit/httpBinding-test.js#L192)
  
2. Coppy tests that are NGSI agnostic to V2 folder
	- startup-test.js
	- ultralightMeasures-test.js
	- ultralightCommands-test.js

3. Implement missing tests in V2
	- configurationsMqtt-test.js

**Comment by fgalan** [comment]
> Im going to summarize all changes needed to have the same coverage (or, at least, the same "scenarios") on V1 and V2 for an effective v1 removal from IoTA Node Lib codebase. Not all of them apply to this specific PR (since still remains some tests on V1 folder)

PR https://github.com/telefonicaid/iotagent-ul/pull/499 adds missing NGSIv2 tests.

I think this PR is now ready to be merged. @jason-fox I understand that after this PR an equivalent of PR https://github.com/telefonicaid/iotagent-json/pull/561 ("Switch tests to NGSI-v2") will be done, right?

**Review by fgalan — APPROVED** [review]
LGTM

Passing the ball to @mapedraza for final LGTM and merge

**Review by mapedraza — APPROVED** [review]
LGTM too

**Comment by fgalan** [comment]
> I think this PR is now ready to be merged. @jason-fox I understand that after this PR an equivalent of PR [telefonicaid/iotagent-json#561](https://github.com/telefonicaid/iotagent-json/pull/561) ("Switch tests to NGSI-v2") will be done, right?

I have discussing this with @mapedraza and it seems that the ""Switch test to NGSI-v2" PR is not needed in this case. However, it seems some NGSI-v1 tests are pending from removal. PR https://github.com/telefonicaid/iotagent-ul/pull/505 should fix that.

**Pull request merged**